Libraries/LibGfx/Painter.h

```diff
@@ -30,6 +31,7 @@ class Painter {
     virtual void stroke_path(Gfx::Path const&, Gfx::Color, float thickness) = 0;
     virtual void stroke_path(Gfx::Path const&, Gfx::Color, float thickness, float blur_radius, Gfx::CompositingAndBlendingOperator compositing_and_blending_operator) = 0;
     virtual void stroke_path(Gfx::Path const&, Gfx::PaintStyle const&, ReadonlySpan<Gfx::Filter>, float thickness, float global_alpha, Gfx::CompositingAndBlendingOperator compositing_and_blending_operator) = 0;
```

Comment:
Does this have any other callers?

Reply:
Yes, it's called from https://github.com/LadybirdBrowser/ladybird/blob/master/Libraries/LibGfx/ImageFormats/TinyVGLoader.cpp#L492

```cpp
                    painter.stroke_path(draw_path, style, {}, command.stroke_width, 1.0f, CompositingAndBlendingOperator::SourceOver);
```

Reply:
@gmta Is TinyVGLoader intentionally still around in this repo?

Reply:
Yes, I believe it's used for rendering the UI icons.